main.go

```diff
@@ -22,6 +22,7 @@ var (
 	host     = flag.String("h", "", "Server hostname")
 	certFile = flag.String("c", "", "TLS cert file")
 	keyFile  = flag.String("k", "", "TLS key file")
+	setName	 = flag.String("o", "", "ConfigurationSet name")
```

Comment:
I would prefer "Amazon SES Configuration Set Name" as description.
I would also prefer to use another flag character, as "o" often stands for "output".
Unfortunately, "c", "s" and "n" are all already taken.
How about "r" as a reference to the rules defined in a configuration set?

Reply:
Thanks @blueimp!
I've made the adjustment to the flag declaration.